new-site/src/docs/foundation/guidelines/localisation.mdx

```diff
@@ -1,10 +1,11 @@
 ---
-slug: "/guidelines/localisation"
+slug: "/foundation/guidelines/localisation"
 title: "Localisation"
+nav_title: "Localisation"
 ---
 
-import LeadParagraph from '../../components/LeadParagraph';
-import Image from '../../components/LeadParagraph';
+import LeadParagraph from '../../../components/LeadParagraph';
+import Image from '../../../components/LeadParagraph';
 
 # Localisation
 
```

Comment:
Navigation component link on this page is still pointing into elements folder

Reply:
- [ ] Fixed a broken link in the Localisation guideline page